shared/js/angularProjects.js

```diff
@@ -45,6 +45,10 @@ services.factory('projectsService', ['$resource',
       }
     };
 
+    Project.prototype.isPublishableProjectType = function () {
+      return (this.projectType !== 'weblab');
+    };
```

Comment:
This would be better as a whitelist -- only allow the types that we show in the public gallery, i.e. applab, gamelab, artist or playlab.